components/bitcoin-topics/layouts/TopicBanner.tsx

```diff
@@ -82,6 +197,23 @@ export default function TopicBanner({
         return () => window.removeEventListener("scroll", handleScroll)
     }, [])
 
+    useEffect(() => {
+        const handleCompletionChange = () => {
+            const saved = localStorage.getItem("completedTopics")
+            if (saved) {
+                setCompletedTopics(JSON.parse(saved))
```

Comment:
There is no `completedTopics` State or setter `setCompletedTopics`

Reply:
fixed